package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "discovery",
   "description": "App to view, filter and edit CDF data and relationships.",
-  "version": "0.1.3",
+  "version": "0.1.4",
```

Comment:
Hmm? PR title is 0.1.3

Reply:
Yea was gonna re ping you for this because we should bump this earlier so staging is always 1 version higher than prod.